definitions/pipelines/somatic_wgs.cwl

```diff
@@ -0,0 +1,529 @@
+            omni_vcf: omni_vcf
+            intervals: qc_intervals ##todo
+            picard_metric_accumulation_level: picard_metric_accumulation_level
+            bqsr_intervals: bqsr_intervals
+            minimum_mapping_quality: qc_minimum_mapping_quality
+            minimum_base_quality: qc_minimum_base_quality
+            per_base_intervals: per_base_intervals
+            per_target_intervals: per_target_intervals
+            summary_intervals: summary_intervals
+            final_name:
+                source: tumor_name
+                valueFrom: "$(self).bam"
+        out:
+            [bam, mark_duplicates_metrics, insert_size_metrics, insert_size_histogram, alignment_summary_metrics, gc_bias_metrics, gc_bias_metrics_chart, gc_bias_metrics_summary, wgs_metrics, flagstats, verify_bam_id_metrics, verify_bam_id_depth, per_base_coverage_metrics, per_base_hs_metrics, per_target_coverage_metrics, per_target_hs_metrics, summary_hs_metrics, bamcoverage_bigwig] 
+    normal_alignment_and_qc:
+        run: wgs_alignment.cwl
+        in:
+            reference: reference
+            bams: normal_bams
+            readgroups: normal_readgroups
+            mills: mills
+            known_indels: known_indels
+            dbsnp_vcf: dbsnp_vcf
+            omni_vcf: omni_vcf
+            intervals: qc_intervals ##todo
+            picard_metric_accumulation_level: picard_metric_accumulation_level
+            bqsr_intervals: bqsr_intervals
+            minimum_mapping_quality: qc_minimum_mapping_quality
+            minimum_base_quality: qc_minimum_base_quality
+            per_base_intervals: per_base_intervals
+            per_target_intervals: per_target_intervals
+            summary_intervals: summary_intervals
+            final_name:
+                source: normal_name
+                valueFrom: "$(self).bam"
+        out:
+            [bam, mark_duplicates_metrics, insert_size_metrics, insert_size_histogram, alignment_summary_metrics, gc_bias_metrics, gc_bias_metrics_chart, gc_bias_metrics_summary, wgs_metrics, flagstats, verify_bam_id_metrics, verify_bam_id_depth, per_base_coverage_metrics, per_base_hs_metrics, per_target_coverage_metrics, per_target_hs_metrics, summary_hs_metrics, bamcoverage_bigwig] 
+    concordance:
+        run: ../tools/concordance.cwl
+        in:
+            reference: reference
+            bam_1: tumor_alignment_and_qc/bam
+            bam_2: normal_alignment_and_qc/bam
+            vcf: somalier_vcf
+        out:
+            [somalier_pairs, somalier_samples]
+    detect_variants:
+        run: detect_variants.cwl
+        in:
+            reference: reference
+            tumor_bam: tumor_alignment_and_qc/bam
+            normal_bam: normal_alignment_and_qc/bam
+            interval_list: interval_list
+            dbsnp_vcf: dbsnp_vcf
+            cosmic_vcf: cosmic_vcf
+            panel_of_normals_vcf: panel_of_normals_vcf
+            strelka_exome_mode:
+                default: true
```

Comment:
I don't know enough about how Strelka works; is exome mode appropriate for this WGS workflow?

Reply:
I changed it to false. According to strelka, from what i understood, the interval list is what matters. If we specify the exome_mode as true, it would subset the list to only exome region. So making it false should work on the whole genome.